tests/test_core/test_new_alphabet.py

```diff
@@ -0,0 +1,124 @@
+import numpy
+import pytest
+
+from cogent3.core.new_alphabet import (
```

Comment:
noting methods/functions etc. need to be removed from imports and to reference modules in-line